envi/cli.py

```diff
@@ -372,17 +373,7 @@ def do_config(self, line):
             return self.do_help('config')
 
         if len(args) <= 0 and not options.do_save:
-            #FIXME for now we will hard code one level of sections
-            subnames = self.config.getSubConfigNames()
-            subnames.sort()
-            for subname in subnames:
-                subcfg = self.config.getSubConfig(subname)
-                options = list(subcfg.keys())
-                options.sort()
-                for optname in options:
-                    optval = subcfg.get(optname)
-                    self.vprint('%s.%s=%s' % (subname, optname, json.dumps(optval)))
-
+            self.vprint(self.config.reprConfigPaths())
```

Comment:
this not only is code-reuse, but the listing of paths/options is more complete.  the original only went one level deep.